packages/oid4vci-holder/__tests__/shared/MetadataMocks.ts

```diff
@@ -0,0 +1,869 @@
+    authorizationServerMetadata: {
+      authorization_endpoint: 'https://auth.research.identiproof.io/oauth2/authorize',
+      code_challenge_methods_supported: ['S256'],
+      grant_types_supported: ['authorization_code', 'urn:ietf:params:oauth:grant-type:pre-authorized_code', 'client_credentials', 'refresh_token'],
+      introspection_endpoint: 'https://auth.research.identiproof.io/oauth2/introspect',
+      introspection_endpoint_auth_methods_supported: ['client_secret_basic', 'client_secret_post', 'client_secret_jwt', 'private_key_jwt'],
+      issuer: 'https://auth.research.identiproof.io',
+      jwks_uri: 'https://auth.research.identiproof.io/oauth2/jwks',
+      response_types_supported: ['code'],
+      revocation_endpoint: 'https://auth.research.identiproof.io/oauth2/revoke',
+      revocation_endpoint_auth_methods_supported: ['client_secret_basic', 'client_secret_post', 'client_secret_jwt', 'private_key_jwt'],
+      token_endpoint: 'https://auth.research.identiproof.io/oauth2/token',
+      token_endpoint_auth_methods_supported: ['client_secret_basic', 'client_secret_post', 'client_secret_jwt', 'private_key_jwt'],
+    },
+    authorizationServerType: 'OID4VCI',
+    authorization_endpoint: 'https://auth.research.identiproof.io/oauth2/authorize',
+    authorization_server: 'https://auth.research.identiproof.io',
+    credentialIssuerMetadata: {
+      authorization_server: 'https://auth.research.identiproof.io',
+      credential_endpoint: 'https://issuer.research.identiproof.io/credential',
+      credentials_supported: {
+        'Cyber Security Certificate': {
+          formats: {
+            jwt_vc: {
+              cryptographic_binding_methods_supported: ['did'],
+              cryptographic_suites_supported: ['ES256'],
+              types: ['VerifiableCredential', 'Cyber Security Certificate'],
+            },
+          },
+        },
+        OpenBadgeCredential: {
+          formats: {
+            jwt_vc: {
+              cryptographic_binding_methods_supported: ['did'],
+              cryptographic_suites_supported: ['ES256'],
+              types: ['VerifiableCredential', 'OpenBadgeCredential'],
+            },
+          },
+        },
+        OpenBadgeExtendedCredential: {
+          formats: {
+            jwt_vc: {
+              cryptographic_binding_methods_supported: ['did'],
+              cryptographic_suites_supported: ['ES256'],
+              types: ['VerifiableCredential', 'OpenBadgeExtendedCredential'],
+            },
+          },
+        },
+      },
+      issuer: 'https://issuer.research.identiproof.io',
+      jwks_uri: 'https://issuer.research.identiproof.io/.well-known/did.json',
+    },
+    credential_endpoint: 'https://issuer.research.identiproof.io/credential',
+    issuer: 'https://issuer.research.identiproof.io',
+    token_endpoint: 'https://auth.research.identiproof.io/oauth2/token',
+  },
+}
+
+export const GET_INITIATION_DATA_PRE_AUTHORIZED_OPENID_INITIATE_ISSUANCE = {
```

Comment:
No initiate issuance. Called credential offer

Reply:
Changed to credential_offer